### ISSUE TITLE
Documentation missing for adding releases to Yugabyte Platform

### ISSUE THREAD

**Issue by dave-philp** [yugabyte/yugabyte-db#6369]
The documentation currently describes how to [deploy updates to universes](https://docs.yugabyte.com/latest/yugabyte-platform/manage-deployments/upgrade-software/) using Yugabyte Platform, but the process for adding a release is ambiguous.

We were able to deduce how to do this with a bit of guesswork, but a clear process for doing so would be very helpful.

**Comment by polarweasel** [comment]
Is this asking for the process by which a Platform administrator can update the list of available YugabyteDB releases that they can deploy on their nodes? @lizayugabyte can you check into this and make sure the docs are current?

**Comment by lizayugabyte** [comment]
@dave-philp I'm not sure what "adding a release" means here. Is it upgrading Yugabyte Platform to a new version?
Here's the latest "Upgrade Software" dialog:
![Screen Shot 2021-12-08 at 1 04 06 PM](https://user-images.githubusercontent.com/77016159/145260542-b64da3ae-31c2-47a8-96c3-f45c8f707cb6.png)

**Comment by dave-philp** [comment]
From memory this was related to a bug in a much earlier release than the current one, where we had to manually download the upgrade files to make them available in the "Server Version" picklist. I spoke to one of the YB team about it not long after and they indicated it was a bug.

We're no longer using Yugabyte so I can't confirm if this is still an issue, my apologies @lizayugabyte. If nobody else has reported it I would suggest it has been resolved.
